src/Event/Http/HttpResponse.php

```diff
@@ -16,11 +16,15 @@ final class HttpResponse
     /** @var string */
     private $body;
 
-    public function __construct(string $body, array $headers = [], int $statusCode = 200)
+    /** @var float|null */
+    private $payloadVersion;
+
+    public function __construct(string $body, array $headers = [], int $statusCode = 200, ?float $payloadVersion = 1.0)
```

Comment:
I think we should avoid passing `$payloadVersion` in the constructor: this class represents a HTTP response, and that's independent from the version of API Gateway.

However, when we convert the response to API Gateway -> that's where we need to know. I suggest to pass `$payloadVersion` as a parameter to `toApiGatewayFormat()` instead (just like `$multiHeaders`).

That should also simplify the places where this constructor is called. Since this constructor can also be called by Bref users now, it's a good thing as well to remove the parameter from the constructor (they shouldn't care about this parameter in their code).